src/GeneralStateTestsFiller/stPreCompiledContracts/delegatecall09UndefinedFiller.yml

```diff
@@ -0,0 +1,46 @@
+delegatecall09Undefined:
+  # This test invokes delegatecall to the Blake2b (09) precompile in the revision it has been
+  # defined (Istanbul) and in the one before it (ConstantinopleFix). For the defined precompile
+  # the call should result in a failure and the single gas unit provided should be consumed.
+  # In case the precompile is undefined, the call should succeed and no gas should be consumed.
+  env:
+    currentCoinbase: a94f5374fce5edbc8e2a8697c15331677e6ebf0b
+    currentDifficulty: 1
+    currentGasLimit: 1000000
+    currentNumber: 1
+    currentTimestamp: 1000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  expect:
+    - indexes:
+        data: !!int -1
+      network:
+        - ConstantinopleFix
+        - Istanbul
+      result:
+        0000000000000000000000000000000000000009:
+          shouldnotexist: 1
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      nonce: 0
+      balance: 100000000000000
+      storage: {}
+      code: ''
+    095e7baea6a6c7c4c2dfeb977efac326af552d87:
+      nonce: 0
+      balance: 0
+      storage: {}
```

Comment:
but since 0 is expected lets initiate the storage of 0 with 0xff or smth non zero

Reply:
Added. Storage values is expected to be 1 (because delegatecall to empty account succeeds).